spec/policies/response_template_policy_spec.rb

```diff
@@ -62,6 +62,6 @@ def valid_attributes
     let(:response_template) { create(:response_template, type_of: "mod_comment", user: nil) }
 
     it { is_expected.to permit_actions(%i[moderator_index moderator_create]) }
-    it { is_expected.not_to permit_actions(%i[create update destroy]) }
+    it { is_expected.to forbid_actions(%i[admin_create update destroy]) }
```

Comment:
Unrelated, but rspec told me that `not_to permit_actions` can produce weird results, so I switched to `forbid_actions`. Also `create` is actually allowed in the policy.